src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -49,6 +49,11 @@ public boolean isEvent() {
         return type == Type.EVENT;
     }
 
+    public String getStringType() {
+        String[] types = new String[] {"", "event-", "mod-"};
+        return types[type.ordinal()];
```

Comment:
Similar issue. `"event-"` and `"mod-"` repeated below in `parseTagType`. Better to set as variables. 😄